github/ci/prow/templates/cherrypicker_deployment.yaml

```diff
@@ -34,13 +34,17 @@ spec:
         image: gcr.io/k8s-prow/cherrypicker:v20200623-9f5410055c
         imagePullPolicy: Always
         args:
+        - --config-path=/etc/config/config.yaml
         - --dry-run=false
         - --github-endpoint=http://ghproxy
         - --github-endpoint=https://api.github.com
         ports:
           - name: http
             containerPort: 8888
         volumeMounts:
+        - name: config
```

Comment:
This looks good.

Reply:
But we got an error that the -config-path was an argument that was not expected but provided. Meh.

Reply:
In the docs they say that it's required now.